code/src/utils/parse-product-description/parseProductDescription.tsx

```diff
@@ -0,0 +1,4 @@
+const parseProductDescription = (description: string) =>
+  description.split("/").join("\n");
+
+export default parseProductDescription;
```

Comment:
In `ProductDetailsContainer` logic was different. There was a paragraph for each separated word

![image](https://github.com/user-attachments/assets/ddb99ea5-4a18-47ce-ad6a-df74d27c44a5)

I suggest adding additional `map` to it

Reply:
In my approach I used `\n` to make new line. After a little investigation, I decided to use your approach, because it is slightly better for accessibility and semantics. I created `ProductDescription` component for it and used it in both files.